examples/fscloud/main.tf

```diff
@@ -10,6 +10,18 @@ module "resource_group" {
   existing_resource_group_name = var.resource_group
 }
 
+##############################################################################
+# Key Protect Instance
+##############################################################################
+resource "ibm_resource_instance" "key_protect_instance" {
+  name              = "${var.prefix}-key-protect-instance"
+  resource_group_id = module.resource_group.resource_group_id
+  service           = "kms"
+  plan              = "tiered-pricing"
+  location          = "us-south"
+  service_endpoints = "public-and-private"
```

Comment:
should be private only if this is fscloud example

Reply:
done